src/components/organisms/details-collapsible/index.tsx

```diff
@@ -0,0 +1,46 @@
+import * as RadixCollapsible from "@radix-ui/react-collapsible"
+import clsx from "clsx"
+import React, { useState } from "react"
+import ArrowDownIcon from "../../fundamentals/icons/arrow-down-icon"
+import ArrowUpIcon from "../../fundamentals/icons/arrow-up-icon"
+
+type DetailsCollapsibleProps = {
+  rootProps?: RadixCollapsible.CollapsibleProps
+  triggerProps?: RadixCollapsible.CollapsibleTriggerProps
+  contentProps?: RadixCollapsible.CollapsibleContentProps
+  children: React.ReactNode
+}
```

Comment:
Let me know, what you think of this @pKorsholm 😄

Reply:
I think it's good 😄  just so we have some control from outside the component as well